docs/source/notes/distributed_training.rst

```diff
@@ -306,6 +306,61 @@ In `gpt.py <https://github.com/mosaicml/examples/blob/6972fe3000d5a5480d8757ff71
 A very similar auto wrap policy is provided for activation checkpointing, with analogous rule #1 that looks for :code:`module._activation_checkpointing = True | False` and rule #2 that looks for :code:`def activation_checkpointing_fn(module: torch.nn.Module) -> bool`.
 
 
+**Experimental:** Composer enables the users to specify custom FSDP args for all wrapped modules. This is enabled by returning a dictionary of args instead of returning a bool.
+
+.. code:: python
+
+    import torch.nn as nn
+
+    class Block(nn.Module):
+        ...
+
+    class BlockRequiringCustomArgs(nn.Module):
+        ...
+
+    class Model(nn.Module):
+        def __init__(self, n_layers):
+            super().__init__()
+            self.blocks = nn.ModuleList([
```

Comment:
maybe this model should actually contain some `BlockRequiringCustomArgs`?

Reply:
valid point